lib/src/presentation/widgets/sign_in_text_fields.dart

```diff
@@ -19,22 +20,33 @@ class PasswordTextFormField extends StatelessWidget {
   final TextStyle style;
   final InputDecoration decoration;
 
+  @override
+  _PasswordTextFormFieldState createState() => _PasswordTextFormFieldState();
+}
+
+class _PasswordTextFormFieldState extends State<PasswordTextFormField> {
+  bool obscurePassword = true;
```

Comment:
Should make this an optional thing. So the user can set this to true. Need to pass this is with the other style info